### DISCUSSION TITLE
Fraudsters

### DISCUSSION THREAD

**Discussion by Eldorado777** [madMAx43v3r/chia-plotter#93] [General]
In the Russian community in telegram, fraudsters, the owner of this group https://t.me/chiafaq , sell this program for $471!!!!!!! Please, as the author of the program, do something!!! People are deceived and sold other people's products...!!!!

**Comment by Forusim** [comment]
Just give them link to this repo, so they can compile it for themselves or pay the fraudsters for compiling.

**Comment by madMAx43v3r** [comment]
I don't speak russian...

**Comment by SebMoore** [comment]
Yeah found it - the admin tells people that they can pay him 35k rubles for the program. Then people start linking the original repo (here). Then people start accusing the admin of deleting the links. Then lots of people start laughing at the admin. So I don't think many people are going to buy it lol.

**Comment by SebMoore** [comment]
Один плот можно засеять от 25 минут)! Ребята, есть программка с открытым исходным кодом при помощи которой можно настраивать генерацию одного плота за 25 минут. Мой знакомый админ протестил ее, очень долго мучался с танцами и бубнами и у него все сработало. 
Если кому интересно, то мы можем настроить это и вам на Linux (на Винде еще не тестировали это, но тоже можно попробовать). Цена: 35 000 рублей.
Схема работы такая: вы платите мне, админ Вам все делает (через TeamViewer), вы проверяете и если говорите мне, что все ок, то я перевожу деньги ему. Если у вас не сработает, то я верну деньги. Таким образом, я гарант того, что вас не обманут. Кроме того, админ гарантирует бесплатную поддержку в течение 3-х дней после установки и настройки.
Первым 10-ти людям, кто скинет предоплату, скидка 10 т.р. 
Кто заинтересовался, переводите деньги сюда: 4377 7237 4147 2348 (Максим К., Тинькофф) и пишите мне в личку.

TRANSLATED:

One raft can be seeded from 25 minutes)! Guys, there is an open source program with which you can configure the generation of one raft in 25 minutes. My friend, the admin, protested it, suffered for a very long time with dances and tambourines, and everything worked for him.
If anyone is interested, then we can configure this for you on Linux (we have not tested this on Windows yet, but you can also try it). Price: 35,000 rubles.
The scheme of work is as follows: you pay me, the admin does everything for you (through TeamViewer), you check and if you tell me that everything is ok, then I transfer the money to him. If it doesn't work for you, I will refund your money. Thus, I am the guarantor that you will not be deceived. In addition, the admin guarantees free support for 3 days after installation and configuration.
The first 10 people who throw off the prepayment get a discount of 10 tr.
Who is interested, transfer money here: 4377 7237 4147 2348 (Maxim K., Tinkoff) and write to me in a personal.

**Comment by SebMoore** [comment]
Dunno what the part about "suffered for a very long time with dances and tambourines" is on about, but it appears his service is that he essentially does the build steps for you. Decent profit for him for running a couple of apt-gets and an sh script!

**Comment by grayfallstown** [comment]
It looks like they sell installation service and technical support, not the program itself. I would not consider this fraud as long as they deliver.

**Comment by SebMoore** [comment]
Yeah exactly - so not really a total _scam_, just more of a ripoff

**Comment by grayfallstown** [comment]
> Yeah exactly - so not really a total _scam_, just more of a ripoff

Considering the technical expertise of most farmers, a guarantee they will do everything they can to get it to run on your server(s) regardless of which linux they run on..  $471 fixed price is actually quite legit. That price is equivalent to four hour of work from experts where I live. Considering you have no idea what kind of servers or how many you have to deal with four hours seems fine for larger farmers.

Or they just drop the binaries they compiled before and hope for the best. If it does not run, you get a refund and they focus on the next customer. That would be crap.

**Comment by SebMoore** [comment]
Yeah I suppose so - it just seemed to be targeted towards more ameteur people who would jump at the idea of "faster plots = more money". If it was in some kind of professional telegram, then fine. I'd take 471 as a good price. But I'm not convinced that this is very professional, and I do fear it's more of a "here's my precompiled binary have a nice day" sort of thing.

**Comment by Andrey768** [comment]
There is no fraud in favor of this telegram channel, all proven guys are great

**Comment by Hardcore-fs** [comment]
you know..... for socialists they are really effective capitalists....
The real problem is all the people who fall for "get rich quick" schemes......
oh oh....... i can get rich in bitcoin.... chia... tulips...{insert scheme here}
If dumb-asses want to part with their money without doing minimum research "Darwin" will take care of it.
